.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Thanks for trying out Zero to JupyterHub!
+
+If you're having issues with your deployment, it is
+really helpful if you include the contents of your
+`config.yaml` file with your issue. We've put
+a place below where you can paste this in. Thanks!
+
+Make sure you remove all sensitive information such as:
+
+    any special keys under auth
```

Comment:
Make this a bulleted list?

Also we should emphasize this more. People posting [secure keys on GitHub](https://www.bloomberg.com/news/articles/2017-11-22/uber-hack-shows-vulnerability-of-software-code-sharing-services) is what got Uber to lose 50+ million people's private info + pay hackers 100k, so let's be very careful in asking people to post this kinda stuff.